python/apidaora/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  github: dutradda/apidaora
+  version: 0.11.4
```

Comment:
please use X.Y format, since `0.11.8` (example) could be installed according to requirements